src/frontend/dpia-webapp/README.md

```diff
@@ -0,0 +1,46 @@
+# dpia-webapp
+
+This template should help get you started developing with Vue 3 in Vite.
+
+## Recommended IDE Setup
```

Comment:
Generally not a big fan of IDE specifics. But I also realize 99.999% use Visual studio in BC-gov so my comment is irrelevant.